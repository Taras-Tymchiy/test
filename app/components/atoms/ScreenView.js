// @flow
import * as React from 'react';
import {
    StyleSheet,
    View,
} from 'react-native';
import { COLOR_SCREEN_BG } from '../styles/constants';
import StyledHOC from '../HOCs/StyledHOC';

const styles = StyleSheet.create({
  view: {
    flex: 1,
    backgroundColor: COLOR_SCREEN_BG,
  },
});

export default StyledHOC(View, styles.view);
