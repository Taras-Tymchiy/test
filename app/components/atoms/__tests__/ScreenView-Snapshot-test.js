import React from 'react';
import renderer from 'react-test-renderer';
import ScreenView from '../ScreenView';
import { Text } from 'react-native';

it('renders a ScreenView using Snapshots', () => {
  expect(renderer.create(
    <ScreenView />
  )).toMatchSnapshot();
});

it('renders an ScreenView with children using Snapshots', () => {
  expect(renderer.create(
    <ScreenView>
      <Text>Test</Text>
    </ScreenView>
  )).toMatchSnapshot();
});
