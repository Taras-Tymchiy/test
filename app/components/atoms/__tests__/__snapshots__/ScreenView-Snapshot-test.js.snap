// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ScreenView using Snapshots 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#EEE",
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`renders an ScreenView with children using Snapshots 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#EEE",
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Test
  </Text>
</View>
`;
